crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -1280,6 +1475,35 @@ unsafe impl<T: Component> WorldQuery for ChangeTrackers<T> {
     }
 }
 
+impl<T: Component, const N: usize, const ALIGN: usize> WorldQueryBatch<N, ALIGN>
+    for ChangeTrackers<T>
+where
+    ComponentTicks: AlignedBatchGat<N, ALIGN>,
+{
+    #[inline]
+    unsafe fn table_fetch_batched_aligned<'w>(
```

Comment:
SAFETY comment needed.

Reply:
The safety constraints are documented in the `WorldQueryBatch`, similar to the safety constraints of `WorldQuery`'s `table_fetch` function -- is there something else that should be added here?

Reply:
I'll defer to other reviewers on that point :)